examples/proto-to-ballerina/proto_to_ballerina.out

```diff
@@ -1,4 +1,4 @@
-# Run the following command in Ballerina tools distribution for mock service and stub generation.
+# Run the following command in Ballerina tools distribution for mock service.
```

Comment:
```suggestion
# Execute the below command in the Ballerina tools distribution to run the mock service.
```

Reply:
This command is not to run the service. It used to generate the mock service